frontend/src/components/elements/Audio/Audio.tsx

```diff
@@ -24,35 +24,22 @@ interface Props {
 }
 
 class Audio extends React.PureComponent<Props> {
-  private audioRef = React.createRef<HTMLAudioElement>()
-
-  public componentDidMount = (): void => {
-    this.updateTime()
-  }
-
-  public componentDidUpdate = (): void => {
-    this.updateTime()
-  }
-
-  private updateTime(): void {
-    if (this.audioRef.current) {
-      const startTime = this.props.element.get("startTime")
-      this.audioRef.current.currentTime = startTime
-    }
-  }
-
   public render(): React.ReactNode {
     const { element, width } = this.props
+    if (element.get("url")) {
+      return (
+        <audio
+          controls
+          src={element.get("url")}
+          className="stAudio"
+          style={{ width }}
+        />
+      )
+    }
     const dataUrl =
       "data:" + element.get("format") + ";base64," + element.get("data")
     return (
-      <audio
-        ref={this.audioRef}
```

Comment:
When you merge `develop`, make sure to keep this `ref` attribute

Reply:
Same here. Please re-add (and also add to the other `<audio>` element, above.)

Reply:
Yeah I am doing that... I don't really understand why this happened though.